010-code-style.md

```diff
@@ -8,6 +8,7 @@ Code formatting used in Yii 3 packages is based on [PSR-1](https://www.php-fig.o
 - Use English only.
```

Comment:
Hm, why this was removed? English code is not good practice?

Reply:
It's not removed. `-` here is list item symbol :)

Reply:
Uch, interface of mobile github app suck.
It shows red background on this and green on added content :-|